flask_security/views.py

```diff
@@ -70,7 +70,7 @@ def login():
     if request.json:
         form = form_class(MultiDict(request.json))
     else:
-        form = form_class()
+        form = form_class(request.form)
```

Comment:
I am not really sure if we need it. Flask-WTF does it for us (https://flask-wtf.readthedocs.io/en/stable/api.html#flask_wtf.FlaskForm), but I can understand that it might be useful for custom form classes.